quick-start/README.md

```diff
@@ -57,6 +56,34 @@ Confirm the plugin is successfully installed by
 
 `$ docker plugin ls`
 
+
+### HPE 3PAR Fibre Channel plugin
+
+Support for HPE 3PAR FC Volume Plugin has been added in hpestorage/hpedockervolumeplugin:2.0 and it is tested against Docker 17.06 EE.
+
```

Comment:
Add a note section: 
Pre-requisite for the FC plugin to work is that the docker host (where the plugin is installed) should be properly zoned to storage array.